src/shared_realm.hpp

```diff
@@ -33,17 +32,19 @@
 #include <memory>
 
 namespace realm {
-class BinaryData;
 class BindingContext;
 class Group;
 class Realm;
 class Replication;
 class SharedGroup;
 class StringData;
+class Table;
```

Comment:
I guess Xcode resolves these differently, since commenting this out still compiled.